Date: Tue, 15 Jul 2008 08:09:29 -0700
From: Josh Triplett <josht@linux.vnet.ibm.com>
Subject: [PATCH] ftrace: Actually call function pointer in ftrace_stop

ftrace_stop used a function pointed as a no-op expression, rather than
actually calling it.

Signed-off-by: Josh Triplett <josh@kernel.org>
Signed-off-by: Thomas Gleixner <tglx@linutronix.de>
---
 kernel/trace/trace.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux-2.6.24.7-rt21/kernel/trace/trace.c
===================================================================
--- linux-2.6.24.7-rt21.orig/kernel/trace/trace.c	2008-10-08 22:25:06.000000000 -0400
+++ linux-2.6.24.7-rt21/kernel/trace/trace.c	2008-10-08 22:25:08.000000000 -0400
@@ -3268,7 +3268,7 @@ void ftrace_stop(void)
 	if (tr->ctrl) {
 		tr->ctrl = 0;
 		if (saved_tracer && saved_tracer->ctrl_update)
-			saved_tracer->ctrl_update;
+			saved_tracer->ctrl_update(tr);
 	}
 
 
